papa/src/main/java/papa/InputTracker.kt

```diff
@@ -34,7 +37,40 @@ interface InputTracker {
   companion object : InputTracker by RealInputTracker
 }
 
-class DeliveredInput<T : InputEvent>(
-  val event: T,
-  val deliveryUptimeMillis: Long
-)
+/**
+ * [InputEventType] is either a [MotionEvent] or a [KeyEvent].
+ */
+class DeliveredInput<InputEventType : InputEvent>(
+  val event: InputEventType,
+  val deliveryUptime: Duration,
+  val framesSinceDelivery: Int,
+  private var endTrace: ((() -> Unit))?
+) {
+
+  val eventUptime: Duration
+    get() = event.eventTime.milliseconds
```

Comment:
Nit: Does this need to be a custom getter, or can it just be initialized? Can `event.eventTime.milliseconds` change?

Reply:
It won't change, but as a general practice I avoid creating backing fields when delegation works just fine and is cheap, especially if the backing field might never be actually read.